popup/OpenClosed.md

```diff
@@ -0,0 +1,101 @@
+                                                Current_t==12 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                    Current_t==13 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                        Current_t==14 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                            Current_t==15 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                                Current_t==16 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                                    Current_t==17 && calcHour =="13" || CalcHour == "24", "Open",iif(
+                                                                        Current_t==18 && calcHour =="13" || CalcHour == "24", "Open",iif(
+                                                                            Current_t==19 && calcHour != "24", "Closed",iif(
+                                                                                Current_t==20 && calcHour != "24", "Closed",iif(
+                                                                                    Current_t==21 && calcHour != "24", "Closed",iif(
+                                                                                        Current_t==22 && calcHour != "24", "Closed",iif(
+                                                                                            Current_t==0 && calcHour != "23", "Closed", "Call")))))))))))))))))))))));
+
+#if day or hours is other, call the park/store to find out when they are open
+var Status = iif(
+    Day_open== "Other" && CalcHour =="Open", "Call " + $feature.PHONE +" for hours of operation",iif(
+		CalcHour == "Other" && Day_open == "Open", "Open Today. Call " + $feature.phone + " for hours", iif(
+			Day_open == "Open" && Hour_open == "Open", "Currently Open",iif(
+			    CalcDay == "Other" && CalcHour == "Other", "Call " + $feature.PHONE +" for hours of operation",iif(
+			        Hour_open == "Closed", "Curently Closed", iif(
+			            Day_open == "Closed", "Currently Closed", "Call " + $feature.phone + " for hours" ))))));
+
+Return Status}
+
+#apply function to your features
+Open($feature.OPERDAYS,$feature.OPERHOURS)
+
+```
+
+## HTML Template
+
+Next you will need to configure the pop-up with a custom attribute display. Click the View HTML source button and add the following HTML table. You will want to update the expression references to correspond to the 4 expressions you created above. You will also need to update the project phase names to correspond to the different stages in your project phase field. You can add additional cells to each of the rows if you support more phases, for example you would replace this:
+
+```html
+
+```
+
+## Example output
+
+[![open](./images/oepn.png)]
```

Comment:
So you're not allowed to upload an image? If you aren't, then just send it over to me, and I'll add it myself when this PR merges.